### DISCUSSION TITLE
remix is not working

### DISCUSSION THREAD

**Discussion by krishno16** [Cyfrin/foundry-full-course-cu#2375] [General]
![Screenshot 2024-09-03 141516](https://github.com/user-attachments/assets/c3e3e3bb-c1d8-44f4-b21f-46d350aba052)
can you please help me why  can't open the files after clicking

**Comment by EngrPips** [comment]
Hello @krishno16, Please refresh your browser.

**Reply by krishno16** [reply]
it won't work i am clicking continously but the files are not open 
![Screenshot 2024-09-03 235645](https://github.com/user-attachments/assets/4013aa4f-7fe7-4b1a-befc-284d69d4da73)

**Reply by EngrPips** [reply]
When I said refresh your browser, I meant click on the icon I circled in the image below. Then, retry opening the file. 

<img width="1440" alt="Screenshot 2024-09-03 at 21 12 30" src="https://github.com/user-attachments/assets/a46a1ee9-fc28-4d3b-ac57-013d4fd00fd8">
